docs/DeveloperGuide.adoc

```diff
@@ -753,19 +746,19 @@ is `matric` , it calls `model` 's `sortModulesByMatricNumber()` command.
 [[SessionView]]
 === Session View
 
-Session View is the term used to characterise the different functionalities related to
-sessions that the user is affiliated with.
+*Session View* is the term used to call the view that contains a list of all sessions
+that haven't been completed yet.
 
 ==== Model Framework
 
-The following class diagrams hows how different classes are related in the functioning
-of the Session View.
+The following class diagram shows how different classes are related in the functioning
+of the *Session View* alongside the classes used in *Student View*.
 
-.Class Diagram of Session View
-image::SessionModelClassDiagram.png[]
+.Model Components - Class Diagram
+image::ModelComponentsClassDiagram.png[]
```

Comment:
This Diagram should have been kept above since it contains the **overall** class diagram . The class diagram for session view shouldn't contain details about the `group `and `student`, since those are not being used in the session view

Could move this back up and say "The following class diagram shows how all the classes are related model framework" ? Alternatively, could add this into the **model** section.

Also the current `SessionModelClassDiagram` looks wonky, I know. Not sure who edited it but ill fix it later!

Reply:
Ahh ok, I'll fix the session one. I felt that this diagram was kinda redundant since literally the next diagram was so similar to it. I suppose can move it up.

Reply:
I moved the diagram up to model.